usecases/optimization_paper/tex/optimization_paper_bibliography.bib

```diff
@@ -0,0 +1,147 @@
+%% Journal article
+@article{bib1,

```

Comment:
I usually add the name of the first author and year (so \cite{Tamsen2023}, potentially adding the abbreviated journal name. This makes reading that in the source code a little easier and also allows to reuse one bib in another one.

Reply:
This is still just part of the template, will be improved.